src/core/atomicdex/constants/dex.constants.hpp

```diff
@@ -8,4 +8,25 @@ namespace atomic_dex
     inline const std::string g_primary_dex_coin{DEX_PRIMARY_COIN};
     inline const std::string g_second_primary_dex_coin{DEX_SECOND_PRIMARY_COIN};
     inline const std::vector<std::string> g_default_coins{g_primary_dex_coin, g_second_primary_dex_coin};
-}
\ No newline at end of file
+    inline const std::vector<std::string> g_wallet_only_coins{
+        "ARRR-BEP20",
+        "RBTC",
+        "NVC",
+        "PAXG-ERC20",
+        "USDT-ERC20",
+        "BET",
+        "BOTS",
+        "CRYPTO",
+        "DEX",
+        "HODL",
+        "JUMBLR",
+        "MGW",
+        "MSHARK",
+        "PANGEA",
+        "REVS",
+        "SUPERNET",
+        "XPM",
+        "XVC",
```

Comment:
XVC should not be set wallet_only, it works fine
XVC-OLD was the one that was not working and they did a chain swap

Reply:
thanks, removed.